### PR TITLE
264 project detail page headings

### LINKED ISSUE #264: Project Details page - headings
The "Project Details" page has 4 headings: "Roles", "Description", "Automatic Settings", and "Provenance"

These changes apply to the show and edit pages (`app/views/projects/show.html.erb` and `app/views/projects/_edit_form.html.erb`)

The only value that we have for "Automatic Settings" at this point is the `mediaflux id`. 

Display the `created_by/on` and `updated_by/on` under the Provenance section.

### PR DESCRIPTION
**Pull request by hectorcorrea**
Adds the `created_by/on` and `updated_by/on` to the "Provenance" section of the Show page. 

We had to do a little bit of refactoring to the code since @JaymeeH noticed that we were not persisting these values in the `medata_json` field.

Closes #264

### CODE REVIEW

File: app/controllers/projects_controller.rb [hunk 1]
```diff
@@ -66,10 +66,26 @@ def user_list_params(counter, key_prefix)
       users.compact.uniq
     end
 
-    def form_metadata
+    def project_timestamps(project:)
+      timestamps = {}
+      # TODO: Use the correct dates instead of the hard coded ones
```
Comment (hectorcorrea) [inline app/controllers/projects_controller.rb:71]: We should now remove this comment

File: app/controllers/projects_controller.rb [hunk 2]
```diff
@@ -66,10 +66,26 @@ def user_list_params(counter, key_prefix)
...
+      if project.nil?
+        timestamps[:created_by] = current_user.uid
+        timestamps[:created_on] = DateTime.now.strftime("%d-%b-%Y %H:%M:%S")
```
Comment (hectorcorrea) [inline app/controllers/projects_controller.rb:74]: Awesome! I am glad you were able to figure out the right incantation for this format!

File: app/views/projects/show.html.erb [hunk 3]
```diff
@@ -39,14 +39,29 @@ Project Details:
   </p>
 </div>
 
+<%
+=begin%>
+  INSERT 2 DIVS. ONE FOR AUTOMATIC SETTINGS -- CONTAINING THE MEDIAFLUX ID -- AND ONE FOR PROVENANCE -- CONTAINING CREATEDBY/ON AND UPDATED BY/ON -- BOTH SHOULD BE ACCESSIBLE VIA  `project.metadata`
+<%
+=end%>
+
```
Comment (hectorcorrea) [inline app/views/projects/show.html.erb:47]: ???

File: app/controllers/projects_controller.rb (hunk 1 above)
Comment (JaymeeH) [reply app/controllers/projects_controller.rb:71]: Will fix

File: app/controllers/projects_controller.rb (hunk 2 above)
Comment (JaymeeH) [reply app/controllers/projects_controller.rb:74]: 🎉🎉

**Review by JaymeeH — APPROVED** [review]
Closes #264

**Pull request merged**